01/ronaldokun/wordvalue.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+from data import DICTIONARY, LETTER_SCORES
+
+char = re.compile(r"[A-Z]{1}")  # I want to match individual UPPER CASE LETTERS
+
+
+def load_words() -> list:
+    """Load dictionary into a list and return list"""
+    with open(DICTIONARY, "r") as words:
+        return words.read().splitlines()
+
+
+def calc_word_value(word: str) -> int:
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    return sum(LETTER_SCORES[c] for c in char.findall(word.upper()))
```

Comment:
always use `get` methods of dictionary if there is a chance that the passed key is not valid. So use `get(key, 0)` to present a valid default case that will not change the sum. Of course, here you use a regex to ensure that all found c are within the dictionary but you never know what happens and as you import DICTIONARY it might change some day